hrpsys_ros_bridge/test/test-samplerobot.py

```diff
@@ -79,15 +79,16 @@ def test_load_pattern(self):
         load_pattern = rospy.ServiceProxy('/SequencePlayerServiceROSBridge/loadPattern', OpenHRP_SequencePlayerService_loadPattern)
         wait_interpolation = rospy.ServiceProxy('/SequencePlayerServiceROSBridge/waitInterpolation', OpenHRP_SequencePlayerService_waitInterpolation)
         basename = rospkg.RosPack().get_path('openhrp3')+'/share/OpenHRP-3.1/sample/controller/SampleController/etc/Sample'
-        tm = 1.0
-        ret = load_pattern(basename = basename, tm = tm)
-        t1 = rospy.get_time()
-        rospy.logwarn("loadPattern = %r %r"%(basename, tm))
-        ret = wait_interpolation()
-        t2 = rospy.get_time()
-        rospy.logwarn("waitInterpolation %f"%(t2-t1))
-        self.assertAlmostEqual(t2-t1, 11, delta=2)
-        #self.assertNotAlmostEqual(trans[1],0,2)
+        if os.path.exists(basename + ".pos")):
```

Comment:
this is synax error, problem is why this passed the test.
It seems,  the test also fails on the travis, https://s3.amazonaws.com/archive.travis-ci.org/jobs/51337147/log.txt, so next problem is why we can not catch this.
does `catkin run_tests --no-status --make-args -j2 -l2 ` returns error cor if it fails?